### ISSUE TITLE
Add pagination to the operations table

### ISSUE THREAD

**Issue by mikevespi** [bcgov/cas-reporting#218]
#### Description:

Blocked by: #217 

The table (grid) showing Operators a list of their Operations needs to be paginated. Confirm page size with designers (make sure it's aligned with how we paginate in other places).

#### Acceptance Criteria:

Given I am an external operator
When I view the list of the operations I have access to
Then that list is displayed in a table
And that table is paginated
And the page size is 20

#### Development Checklist:

- [ ] Enable pagination in the `DataGrid` component setup in #217 
- [ ] Make sure the API for the operation list supports pagination query params
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

- BA doesn't expect more than 5 Operations

**Comment by zoeyli-46** [comment]
implemented, closing.
